packages/nitro-protocol/docs/app-devs/quick-start-dapp.md

```diff
@@ -41,3 +41,17 @@ window.channelProvider.enable();
 If everything is setup correctly, you should see the statechannels wallet UI:
 
 ![](assets/wallet-ui.png)
+
+## App <> Wallet security
+
+Having the wallet served in an iframe from a different domain to the app is an important component from a security standpoint. It means app <> wallet communication is cross-origin and therefore disabled by default, but capable of being selectively enabled on a per-origin (per-app) basis.
+
+Ethereum wallets will typically prompt the user, when triggered by an application, to approve access to the accounts it controls for the domain that application is served from. Wallets should not implicitly trust applications or grant them access to sign away assets, because unknown applications could contain malicious code and put those asssets at risk.
+
+The same is true for our wallet: when the `.enable()` call is made, the user is asked for approval (see above).
+
+There are a few differences between a state channel wallet and an Ethereum wallet, that are important to understand.
+
+Because state channel applications are capable of very rapid throughput of state updates (or "Layer 2 transactions"), it would not be feasible to ask for user approval for each one. It is for this reason that our wallet will create an "ephemeral key" to **silently** sign most of your state updates, instead of using your Ethereum key. (When it comes to releasing assets on chain, however the money still goes to your Ethereum account).
```

Comment:
I would emphasize that this ephemeral key is not expected to ever directly have any funds assigned to it on Ethereum and is strictly used for signatures that can be verified by `ecrecover` in `NitroAdjudicator`